### ISSUE TITLE
CT cannot read property vectorComponents of undefined

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/vector-addition#89]
```
vector-addition : fuzz : built : run
Query: fuzz&memoryLimit=1000
Uncaught TypeError: Cannot read property 'vectorComponents' of undefined
TypeError: Cannot read property 'vectorComponents' of undefined
    at https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:436107
    at Array.forEach (<anonymous>)
    at s.value (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:436074)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:435612
    at s (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:11404)
    at t.value (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:32325)
    at e.value (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:36431)
    at e.value (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:35809)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:555505
    at t.value (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/build/phet/vector-addition_en_phet.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzz&memoryLimit=1000:1027:32325)
id: Bayes Chrome
Approximately 8/6/2019, 11:41:24 AM
vector-addition : fuzz : require.js : run
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: PhetioObject.dispose() call is missing from an overridden dispose method
Error: Assertion failed: PhetioObject.dispose() call is missing from an overridden dispose method
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/assert/js/assert.js:22:13)
    at CurvedArrowNode.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/tandem/js/PhetioObject.js?:142:19)
    at VectorAngleNode.disposeVectorAngleNode (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/common/view/VectorAngleNode.js?:131:26)
    at VectorAngleNode.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/common/view/VectorAngleNode.js?:140:12)
    at VectorAngleNode.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/tandem/js/PhetioObject.js?:141:22)
    at VectorNode.disposeVectorNode (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/common/view/VectorNode.js?:296:19)
    at VectorNode.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/common/view/VectorNode.js?:307:12)
    at VectorNode.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/tandem/js/PhetioObject.js?:141:22)
    at Array.removalListener (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/common/view/SceneNode.js?:243:24)
    at ObservableArray._fireItemRemoved (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/axon/js/ObservableArray.js?:165:18)
id: Bayes Chrome
Approximately 8/6/2019, 11:41:24 AM
vector-addition : xss-fuzz : run
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught TypeError: Cannot read property 'vectorComponents' of undefined
TypeError: Cannot read property 'vectorComponents' of undefined
    at https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/equation/model/EquationVectorSum.js?:109:49
    at Array.forEach (<anonymous>)
    at EquationVectorSum.updateSum (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/equation/model/EquationVectorSum.js?:108:27)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/vector-addition/js/equation/model/EquationVectorSum.js?:61:16
    at listener (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/axon/js/Multilink.js?:41:20)
    at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/axon/js/TinyEmitter.js?:59:55)
    at EnumerationProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/axon/js/Property.js?:258:27)
    at EnumerationProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/axon/js/Property.js?:170:14)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/sun/js/buttons/RadioButtonGroupMember.js?:126:16
    at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1565113284353/axon/js/TinyEmitter.js?:59:55)
id: Bayes Chrome
Approximately 8/6/2019, 11:41:24 AM
```

**Comment by brandonLi8** [comment]
should be fixed with 40676e30498c8de24d52ac0e512104227c511400. Will check to see if it fixes.

**Comment by brandonLi8** [comment]
Looks to be fixed, but a new error popped up :
ector-addition : xss-fuzz : run
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught Error: Assertion failed: PhetioObject.dispose() call is missing from an overridden dispose method
Error: Assertion failed: PhetioObject.dispose() call is missing from an overridden dispose method
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/assert/js/assert.js:22:13)
    at CurvedArrowNode.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/tandem/js/PhetioObject.js?:142:19)
    at VectorAngleNode.disposeVectorAngleNode (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/vector-addition/js/common/view/VectorAngleNode.js?:131:26)
    at VectorAngleNode.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/vector-addition/js/common/view/VectorAngleNode.js?:140:12)
    at VectorAngleNode.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/tandem/js/PhetioObject.js?:141:22)
    at VectorNode.disposeVectorNode (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/vector-addition/js/common/view/VectorNode.js?:296:19)
    at VectorNode.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/vector-addition/js/common/view/VectorNode.js?:307:12)
    at VectorNode.PhetioObject.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/tandem/js/PhetioObject.js?:141:22)
    at Array.removalListener (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/vector-addition/js/common/view/SceneNode.js?:243:24)
    at ObservableArray._fireItemRemoved (https://bayes.colorado.edu/continuous-testing/snapshot-1565123227599/axon/js/ObservableArray.js?:165:18)
id: Bayes Chrome
Approximately 8/6/2019, 2:27:07 PM

**Comment by brandonLi8** [comment]
Should be fixed with e6aedd1ca1665cf3c4adc1ab9fb4697aec8059ff

**Comment by brandonLi8** [comment]
Fixed, closing
